cmd/manager/main.go

```diff
@@ -70,22 +70,22 @@ func main() {
 
 	namespace, err := k8sutil.GetWatchNamespace()
 	if err != nil {
-		log.Error(err, "Failed to get watch namespace")
+		log.Error(err, "failed to get watch namespace")
```

Comment:
This commit should only have changes wrt to empty log messages. The other changes should be done in the appropriate commit that you have created.

Reply:
Not sure, if @aravindhp is saying the same thing but all the changes related to

- Bug fixes in 1st commit
- operator-sdk generated code fixes 2nd commit
- Our operator code(excluding generated commit) fixes in 3rd commit

Reply:
will fix this.